gramps/gen/db/dummydb.py

```diff
@@ -149,23 +153,17 @@ class M_A_M_B(ABCMeta, MetaClass):
 # class DummyDb
 #
 # -------------------------------------------------------------------------
-class DummyDb(
-    M_A_M_B(
-        "NewBaseClass",
-        (
-            DbReadBase,
-            Callback,
-            object,
-        ),
-        {},
-    )
-):
+class DummyDbMeta(M_A_M_B):
```

Comment:
Do you need the extra class? Can you not pass `M_A_M_B` as the metaclass directly?